cmd/promtool/main.go

```diff
@@ -29,11 +29,11 @@ import (
 	"github.com/google/pprof/profile"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/api"
-	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
I tend to agree with what @brian-brazil said, but this alias can actually go.

Reply:
I agree it.